backend/src/openarchiefbeheer/accounts/models.py

```diff
@@ -65,8 +68,9 @@ def get_short_name(self):
         return self.first_name
 
     def get_name_with_username(self):
+        username = self.get_username()
         if self.first_name or self.last_name:
             full_name = self.get_full_name()
-            return f"{full_name.strip()} ({self.username})"
```

Comment:
I assume we don't need strip as the input validation prevents this scenario? Or am I missing something?

Reply:
It's because I hadn't noticed when initially writing this that `self.get_full_name()` already does the `.strip()`!